apps/site/pages/docs/themes/midnight.mdx

```diff
@@ -163,20 +163,7 @@ theme: 'midnight',
 }
 ```
 
-3. Add the [Raleway](https://fonts.google.com/specimen/Raleway?query=raleway) font on `src/fonts/WebFonts.tsx`:
```

Comment:
This is current not implemented on `starter.store`.

Reply:
@hellofanny do we have a task on Jira for this?

Reply:
yes! We [do](https://vtex-dev.atlassian.net/browse/FS-854?atlOrigin=eyJpIjoiNzk2YTYxNTQ5NTIxNGE5OWI2YjgzMDk0NTIzMTU4NjkiLCJwIjoiaiJ9)!